backend/src/API/mailAPI.ts

```diff
@@ -0,0 +1,16 @@
+import transporter from '../nodemailer';
+
+require('dotenv').config();
```

Comment:
I think this is unnecessary since `require('dotenv').config()` already gets called in `firebase.ts`